about.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="about.css" />
+    <title>ArtificialEdge Conference</title>
+  </head>
+  <body>
+    <section class="media show-desktop">
+      <div class="media-container">
+        <a href="#"
+          ><img
+            class="media-icon"
+            src="./assets/icons/facebook-media.svg"
+            alt=""
+        /></a>
+        <a href="#"
+          ><img
+            class="media-icon"
+            src="./assets/icons/twitter-media.svg"
+            alt=""
+        /></a>
+        <p class="media-text lato">English</p>
+        <p class="media-text lato">My Page</p>
+        <p class="media-text lato">Login</p>
+      </div>
+    </section>
+    <header class="desktop-header show-desktop">
+      <nav class="desktop-nav-container">
+        <a class="desktop-nav-logo" href="index.html"
+          ><img src="./assets/img/logo.png" alt="ArtificialEdge logo"
+        /></a>
+        <ul class="desktop-nav-list">
+          <li class="desktop-list-item active lato"><a href="#">About</a></li>
```

Comment:
- [OPTIONAL] I know a user can link to the Home Page by clicking the logo, but since this is the About page, I think it would be better to make this link to the Home page. What do you think? 👍 😎 

` <li class="desktop-list-item active lato"><a href="index.html">Home</a></li>`